### ISSUE TITLE
Randomly crouching

### ISSUE THREAD

**Issue by amogusdestroyer** [MeteorDevelopment/meteor-client#3597]
### Describe the bug

Randomly crouching for 1 tic every 10-20 seconds

### Steps to reproduce

```bash
1. Play minecraft with meteor
2. Join any server or world
3. Starting to randomly crouch
```

### Meteor Version

0.5.3

### Minecraft Version

1.19.4

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Jaranamo1** [comment]
Duplicate of #3113

**Comment by ManInMyVan** [comment]
Fixed in e2854ceb11c621c45d481680af8559c14b992baa
